src/olympia/lib/settings_base.py

```diff
@@ -1759,10 +1752,22 @@ def read_only_mode(env, retry_after=None):
     path('static'),
 )
 
-NETAPP_STORAGE = TMP_PATH
-GUARDED_ADDONS_PATH = os.path.join(ROOT, 'guarded-addons')
+# Path related settings. In dev/stage/prod `NETAPP_STORAGE_ROOT` environment
+# variable will be set and point to our NFS/EFS storage
+# Make sure to check overwrites in conftest.py if new settings are added
+# or changed.
+STORAGE_ROOT = env('NETAPP_STORAGE_ROOT', default=path('storage'))
```

Comment:
could we have the env variable also be named `STORAGE_ROOT`?  It's nice to keep them consistent.

Reply:
That would require some ops work to sync that, I'd make that a separate step tbh, start with the ops part, do our change, remove the old env var.